api/src/test/resources/challenge-identify-first-factor-response.json

```diff
@@ -0,0 +1,122 @@
+{
+  "stateHandle": "025r9Yn758Z-zwhMGDm1saTaW1pVRy4t9oTxM7dLYE",
+  "version": "1.0.0",
+  "expiresAt": "2020-10-29T21:17:28.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "select-authenticator-authenticate",
```

Comment:
yes, it's used in [authenticateIdentifyFirstFactorSuccessTest](https://github.com/okta/okta-idx-java/pull/96/files#diff-51d7e7b8a33e8adfd0326b573a24b85a048453deab58619bcd9dc1b1f987c59aR254-R280)
This is the case with 1 factor enabled.

Reply:
I would assume it would go:
1. https://github.com/okta/okta-idx-java/pull/96/files#diff-e6257c8f18a51e77561988408429c1088e44c20a72b4f32835fcda3d0fc8237dR1
2. https://github.com/okta/okta-idx-java/pull/96/files#diff-23d6012cf3e5e84b16fedc76c54448f86239075c5aded4dba6e4ebbc1283759eR1

Do we have a test covering that scenario?

Reply:
1. [authenticateIdentifyFirstSuccessTest](https://github.com/okta/okta-idx-java/pull/96/files#diff-51d7e7b8a33e8adfd0326b573a24b85a048453deab58619bcd9dc1b1f987c59aR205-R232)
2. [authenticateIdentifyFirstFailTest](https://github.com/okta/okta-idx-java/pull/96/files#diff-51d7e7b8a33e8adfd0326b573a24b85a048453deab58619bcd9dc1b1f987c59aR234-R252)

Reply:
@sergiishamrai-okta I'd recommend reviewing the jacoco code coverage report at `coverage/target/site/jacoco-aggregate/index.html` and navigating to `IDXAuthenticationWrapper` class and ensure all paths are covered (should be green).